docker-hub/access-tokens.md

```diff
@@ -10,12 +10,22 @@ Using personal access tokens provides some advantages over a password:
 * You can investigate when an access token was used last, and disable or delete it if you find any suspicious activity.
 * When logged in with an access token, you can't perform any admin activity on the account, including changing the password.
 
+>**Warning**
+>
+> Treat access tokens like your password and keep them secret. Store your
+> tokens securely (for example, in a credential manager).
+{: .warning }
```

Comment:
Suggest we use "Important" attribute to this as it's not really a warning?

Reply:
Could you update this to use the Important attribute, rather than a warning?

Reply:
I agree, updated in new commit